libs/auth-api-lib/src/lib/delegations/constants/zendesk.ts

```diff
@@ -0,0 +1,5 @@
+export const DELEGATION_TAG = 'umsokn_um_umboð_a_mínum_síðum'
```

Comment:
Does the tag actually contain icelandic characters?

Reply:
yes, I remember you talking about maybe getting them to change the tag in Zendesk, to not include the "mínar síður" part, what was the status on that ?